ucp/utils.py

```diff
@@ -39,3 +45,133 @@ def get_address(ifname=None):
             s.fileno(), 0x8915, struct.pack("256s", ifname[:15])  # SIOCGIFADDR
         )[20:24]
     )
+
+
+def get_closest_net_devices(gpu_dev):
```

Comment:
This is not related to this PR but do you think we could/should start using this over `UCXPY_IFNAME` ?

Reply:
Yes, I was hoping we could begin using it. 
But it properly best if it is possible to overwrite it using `UCXPY_IFNAME`.

Reply:
Makes sense